deployment/deploy-qa.sh

```diff
@@ -24,14 +24,12 @@ NC='\033[0m'
   ./deployment/build-example-html.sh "key_live_feebgAAhbH9Tv85H5wLQhpdaefiZv5Dv" "https://api.stage.branch.io" "https://cdn.branch.io/branch-staging-latest.min.js"
   aws s3 cp example.html s3://branch-cdn/example-staging.html
 
-  ./deployment/build-example-html.sh "key_live_hcnegAumkH7Kv18M8AOHhfgiohpXq5tB" "https://api2.branch.io" "https://cdn.branch.io/branch-latest.min.js"
-  aws s3 cp example.html s3://branch-builds/web-sdk/example.html
```

Comment:
was looking at this more, we did this before my previous pr so i kept it in that pr. But, i dont understand the point of this so i removed it

Reply:
how is the example page going to be uploaded in this new flow?

Reply:
looks like it happens in a later file

Reply:
yeah this was deploying to prod, and also under this web-sdk prefix for some reason